app/src/main/java/com/onegini/mobile/exampleapp/view/activity/DevicesListActivity.java

```diff
@@ -48,7 +48,7 @@
   @BindView(R.id.progress_bar)
   ProgressBar progressBar;
 
-  private Subscription subscription;
+  private CompositeDisposable disposables = new CompositeDisposable();
```

Comment:
There's only one Disposable defined in this class, so we can use `Disposable` rather than `CompositeDisposable`